Makefile.include

```diff
@@ -8,8 +8,11 @@ ifeq (,$(filter $(MATCH_MAKE_VERSION),$(MAKE_VERSION)))
       $(MATCH_MAKE_VERSION) or later.)
 endif
 
+# will evaluate to the absolute path of the Makefile it's evaluated in
+MAKEFILEDIR = $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
+
 # 'Makefile.include' directory, must be evaluated before other 'include'
-_riotbase := $(dir $(lastword $(MAKEFILE_LIST)))
+_riotbase := $(MAKEFILEDIR)
```

Comment:
This is changing `RIOTBASE` to use absolute path instead of relative paths, not sure if this can have an impact..

Reply:
I can drop this one - or make `MAKEFILEDIR` use relative paths.

From my gut feeling absolute paths should be more 'stable' though.

Reply:
Never-mind its overridden to an abspath anyway:

```
override RIOTBASE       := $(abspath $(RIOTBASE))
```